AppCenter/AppCenter.xcodeproj/project.pbxproj

```diff
@@ -1475,6 +1475,10 @@
 		3844FF161E8C261D003E9194 /* Model */ = {
 			isa = PBXGroup;
 			children = (
+				3592ABA61DC90E3600EF4592 /* MSLogger.m */,
+				358F9BC22019531F00B9E22C /* MSLogger.h */,
+				F803BBF11E8E3677004B1E7A /* MSCustomProperties.h */,
+				F803BBF21E8E3677004B1E7A /* MSCustomProperties.m */,
 				3844FF1B1E8C2716003E9194 /* MSDevice.h */,
 				3844FF1A1E8C2716003E9194 /* MSDevice.m */,
 				6E3E2C9D1D35701000B1EE50 /* MSLog.h */,
```

Comment:
@jaelim-ms  I see the `MSLog.h` is under `Model` folder, is this not the correct location?

Reply:
`MSLog` is in `Model` folder when you look at this one in Xcode but physical location of the file isn't in `Model`.

Reply:
I pushed a commit for my comment. This is what I meant.
https://github.com/microsoft/appcenter-sdk-apple/pull/1718/commits/4ff5f05445086ae1e4ef40bcae98aeb8695747ad